packages/components/src/Tile/subcomponents/GenericTile/GenericTile.tsx

```diff
@@ -22,6 +22,7 @@ export type GenericTileProps = {
   titleTag?: AllowedHeadingTags
   metadata?: string
   information?: TileInformation | React.ReactNode
+  infoButtonLabel?: string
```

Comment:
Would be good to add a property description here with the `@default` value called out. Something variant prop below :)

Reply:
Done!

Reply:
Done!